### PR TITLE
zh-trans: /docs/tasks/configure-pod-container/configure-persistent-vo…

### PR DESCRIPTION
**Pull request by AdamDang**
…lume-storage.md

zh-trans: /docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+In your shell, create a `/mnt/data` directory:
+-->
+
+## 在你的节点上创建一个 index.html 文件
+
+打开集群中节点的一个 shell。
+如何打开 shell 取决于集群的设置。
+例如，如果您正在使用 Minikube，那么可以通过输入 `minikube ssh` 来打开节点的 shell。
+
+在shell中，创建一个 `/mnt/data` 目录：
```
[the remark this answers is not in the document]
Comment (AdamDang) [reply content/zh/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md]: done！

File: content/zh/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+access for at least one Node.
+
+Here is the configuration file for the PersistentVolumeClaim:
+-->
+
+## 创建 PersistentVolumeClaim
+
+下一步是创建一个 PersistentVolumeClaim。
+Pod 使用 PersistentVolumeClaim 来请求物理存储。
+在本练习中，您将创建一个 PersistentVolumeClaim，它请求至少 3 gibibytes 容量的卷，该卷至少可以为一个节点提供读写访问。
```
[the remark this answers is not in the document]
Comment (AdamDang) [reply content/zh/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md]: done！

File: content/zh/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+
+下面是 PersistentVolumeClaim 的配置文件：
+
+{{< codenew file="pods/storage/pv-claim.yaml" >}}
+
+<!--
+Create the PersistentVolumeClaim:
+-->
+
+创建 PersistentVolumeClaim
```
[the remark this answers is not in the document]
Comment (AdamDang) [reply content/zh/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md]: Done！

**Comment by Hartmarken348** [comment]
Task config zn

**Pull request merged**